### PR TITLE
CLDR-15580 add pis Pijin

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-15580

- [x] This PR completes the ticket.

### CODE REVIEW

File: common/main/pis.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+			<territory type="SB">Solomon Aelan</territory>
+		</territories>
+		<measurementSystemNames>
+			<measurementSystemName type="metric">metrik</measurementSystemName>
+			<measurementSystemName type="UK">UK</measurementSystemName>
+			<measurementSystemName type="US">US</measurementSystemName>
+		</measurementSystemNames>
+		<codePatterns>
+			<codePattern type="language">Langguis: {0}</codePattern>
+			<codePattern type="script">Raeting kaen: [0]</codePattern>
```
Comment (srl295) [inline common/main/pis.xml:35]:
```suggestion
			<codePattern type="script">Raeting kaen: {0}</codePattern>
```

File: common/main/pis.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+			<codePattern type="territory">Rijin: {0}</codePattern>
+		</codePatterns>
+	</localeDisplayNames>
+	<characters>
+		<exemplarCharacters>[a b d e f g h I j k l m n o p r s t u v w y]</exemplarCharacters>
```
Comment (srl295) [inline common/main/pis.xml:40]:
```suggestion
		<exemplarCharacters>[a b d e f g h i j k l m n o p r s t u v w y]</exemplarCharacters>
```

lowercase i?

File: common/main/pis.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+		<exemplarCharacters type="numbers">[1 2 3 5 6 7 8 9 0 $ # + \- / %]</exemplarCharacters>
```
Comment (srl295) [inline common/main/pis.xml:41]:
```suggestion
		<exemplarCharacters type="numbers">[1 2 3 5 6 7 8 9 0 # + \- / %]</exemplarCharacters>
```

`$` is a currency symbol, not a number or number symbol per se.

**Comment by DavidLRowe** [comment]
@srl295 Looks ready for review

**Review by srl295 — APPROVED** [review]

**Pull request merged**